src/ui/component/transactionListRecent/view.jsx

```diff
@@ -41,7 +41,7 @@ class TransactionListRecent extends React.PureComponent<Props> {
 
         {!fetchingTransactions && !hasTransactions && (
           <div className="card__content">
-            <p className="card__subtitle">{__('No transactions... yet.')}</p>
```

Comment:
Make sure to put "removed vaguely threatening message from transaction list" in the release notes

Reply:
📈